Add IQS041 rule (#2405)
home-assistant/developers.home-assistant: docs/core/integration-quality-scale/rules/iqs041.md

---

> title: "IQS041 - The documentation describes use cases to inspire the end-user in using this integration"

Reviewer: ```suggestion
title: "IQS041 - The documentation describes use cases to illustrate how this integration can be used"
```

---

IQS041 is as follows:

The documentation describes use cases to inspire the end-user in using this integration

## Reasoning

> Sometimes seeing a device or service being integrated in Home Assistant can have people respond "Why would I integrate this?".

Reviewer: ```suggestion
Sometimes, seeing a device or service integrated into Home Assistant can make you wonder, "Why would I integrate this?"
```

> For some integrations this is more straight forward than for others.

Reviewer: ```suggestion
For some integrations, the intended use and benefit are more obvious than for others.
```

> We think enriching the documentation with examples of these use cases can help the end-user understand the value of the integration.

Reviewer: ```suggestion
Use case examples in the documentation can showcase the value of an integration.
```
